study.md

```diff
@@ -19,12 +19,23 @@ material. Instead, digest what you've read and repeat it in your own voice.
 -   [Ember.js - Configuring Ember.js: Specifying the URL Type](https://guides.emberjs.com/v2.4.0/configuring-ember/specifying-url-type/)
 -   [Deploying Ember CLI apps to GitHub Pages](http://osxi.github.io/ember/github/git/2015/09/22/ember-cli-apps-on-github-pages.html)
 
+## Additional Sources
+
+[Gawker's Big Mistake](http://www.webmonkey.com/2011/02/gawker-learns-the-hard-way-why-hash-bang-urls-are-evil/)
+[Quora question on hashbangs](https://www.quora.com/Are-hashbang-URLs-a-recommended-practice)
+
 ## Ember and URLs
 
 In your own words, how Ember "stops breaking the web".
 
 ```md
-<!-- your answer here -->
+The use of hashes in a url severes the connection with the server. When a page loads
+what happens is that javascript tells it to scroll to whatever is being called.
+If javascript were to ever go down, then nothing will render. Ember stops this
+through the use of a router to load certain parts of the page based on the
+url that is passed to it.  When the url is changed given the user interaction,
+Ember will send requests to the server for data that is needed to generate the
+appropriate model and template for the view.
```

Comment:
The reason Ember doesn't "break the web" is that it has multiple URLs, each of which is associated with a consistent application state. IN this way, it kind of acts like an old-school static website, which browsers are very comfortable dealing with.